pmm-app/src/shared/components/Elements/Table/Table.styles.ts

```diff
@@ -66,5 +66,13 @@ export const getStyles = stylesFactory((theme: GrafanaTheme) => {
     checkboxColumn: css`
       width: 20px;
     `,
+    tableClassicExplain: css`
```

Comment:
name `tableClassicExplain` is not clear, what does it mean classic explain?

Reply:
'Explain' as part of the query selected, as listed in the summary table, consists of 'Classic' and 'JSON' parts. As I've been working at the table from 'Classic' part of 'Explain' tab, I chose the naming to follow this belongings: 'tableClassicExplain'.

Reply:
Fixed with name changing and adding comments.